### DISCUSSION TITLE
Slinger hardware requirements, Raspberry Pi?

### DISCUSSION THREAD

**Discussion by smt171** [GerryDazoo/Slinger#523] [General]
Hi all! What are the Slinger hardware requirements?

Is anyone using Slinger on a Raspberry pi?
If so what model pi? How much memory? What size disk (SD or USB drive)? 

Also did you start with Raspbian or is there a distro already updated? Just making sure so I don't replicate work already done. Thanks!

**Comment by smt171** [comment]
Pi2b:
Processor: 900 MHz, quad core processor, supports 64 bit, 
Memory: 1 GB memory
Ethernet only, no wifi. 
Ports: 4 USB 2.0
Graphics: Cortex A7 graphics. 1 full size/regular HDMI port. 
Power: Micro USB (older android type) 5.1v, 2.5a

Pi3b:
Processor: 1.2 GHz, quad core processor, supports 64 bit, 
Memory: 1 GB memory
Ports: 4 USB 2.0
Ethernet port. 
WLAN wireless: 802.11n
Bluetooth 4.1
Graphics: Cortex A53 graphics. 1 micro HDMI port.
Power: Micro USB (older android type) 5.1v, 2.5a

Pi4: 
Processor: 1.5 GHz, quad core processor, supports 64 bit, 
Memory: Available in 1,2,or 4 GB. 
Ports: 2 USB 1.0, 2 USB 3.0 (blue)
Ethernet port
WLAN wireless 802.11ac, 
Bluetooth 5.0. 
Graphics: Cortex A72.  2 micro HDMI ports.
Power: USB-C (newer android type). 5.1v, 3a

**Comment by mdehrlich** [comment]
I use a pi B+ with a 16Gb sd card.   I can run simultaneous streams from 2 slingboxes at 1980X1024 30fps.  Works great.

**Comment by smt171** [comment]
> I use a pi B+ with a 16Gb sd card. I can run simultaneous streams from 2 slingboxes at 1980X1024 30fps. Works great.

Thanks! So the B+ is 700 MHz, 512 mb ram. Awesome so any of these should work. 

With Raspbian? Is there guide on what to do after I get Raspbian installed? I have been readinf the discussions and only found one for Mac.

**Comment by GerryDazoo** [comment]
Read the ReadMe Raspberry Pi section

**Comment by glennybob** [comment]
I had several RPi's on hand so I started with a RPi Zero W. Once I understood the instructions and got it up and running, it is running well local and remote. I was pleasantly surprised! RPi's, if you can find them are very expensive while the Zero W is much lower in price.

**Comment by swilsonnc** [comment]
I bet it would work just fine on a Libre AML-S905X-CC (Le Potato) as well.  I haven't tried it as I don't have any spares laying around.  These are $35 on amazon.

**Comment by glennybob** [comment]
Grand idea!!!
